tests/test_ical.py

```
@@ -400,10 +401,30 @@ def test_unique_uid_generation(self):
+                if not line.startswith("UID:")
+            ]
+
+            file_path_2 = Path(valid_path) / "test_calendar_2.ics"
+            self.us_exporter.save_ics(file_path=file_path_2)
+            content_2 = [
+                line
+                for line in file_path_2.read_text().splitlines()
+                if not line.startswith("UID:")
+            ]
+
+            self.assertEqual(content_1, content_2)
```

Ruff (0.15.2)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)